.tekton/konflux-e2e-tests-pull-request.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: |
       event == "pull_request"
+      && !body.pull_request.draft
```

Comment:
imo we need `!has(body.pull_request)`: https://github.com/konflux-ci/build-definitions/pull/952#discussion_r1572342112